doc/source/user_guide/indexing.rst

```diff
@@ -172,6 +172,16 @@ columns.
       df.loc[:, ['B', 'A']] = df[['A', 'B']].to_numpy()
       df[['A', 'B']]
 
+   However, pandas does not aligns AXES when setting ``Series`` and ``DataFrame`` from ``.iloc``.
+
```

Comment:
```suggestion
   However, pandas does not align AXES when setting ``Series`` and ``DataFrame`` from ``.iloc``
   because ``.iloc`` operates by position.

```